setup.py

```diff
@@ -104,7 +104,8 @@ def check_pkg(python_pkg, distro_pkg):
     extras_require={
         "notebook": [
             "ipython",
-            "jupyterlab"
+            "jupyterlab",
+            "ipywidgets"
```

Comment:
can we add a trailing comma, so the next addition will not need to modify that line as well.

Reply:
Done